install/mysql/plugin_fusioninventory-empty.sql

```diff
@@ -698,7 +698,6 @@ CREATE TABLE `glpi_plugin_fusioninventory_computeroperatingsystems` (
   `id` int(11) NOT NULL AUTO_INCREMENT,
   `name` varchar(255) DEFAULT NULL,
   `comment` text DEFAULT NULL,
-  `operatingsystemarchitectures_id` int(11) NOT NULL DEFAULT '0',
```

Comment:
This should not be removed. As for now, data is present in both core and plugin; this will be fixed in the master branch.